helpers/config.py

```diff
@@ -52,18 +52,51 @@ def verify_settings():
     data['general']['parser_opacity'] = get_setting(
         data['general'].get('parser_opacity', 80),
         80,
-        lambda x: (x > 0 and x <= 100)
+        lambda x: (0 < x <= 100)
```

Comment:
These are just a style preference, I could undo them (you'll see several changed throughout this file). I just changed them out of habit while I was scrolling through and forgot about it until now.